expertise/create_dataset.py

```diff
@@ -35,6 +35,10 @@ def __init__(self, openreview_client, openreview_client_v2, config):
             'no_publications': []
         }
 
+        self.venue_list = openreview_client_v2.get_group('venues').members
```

Comment:
I would make this a set so that checks like `domain in self.venue_list` are O(1) instead of O(n)